### DISCUSSION TITLE
How do you change your Github Repo Description?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#132595] [New to GitHub]
I just made a Repository (pyspeedmath) and I made a typo on the description. How do I edit it?. I looked in the setting button, but just saw "Change Repository Name", and yes, I have checked other discussions, a lot of them. Can you guys help?

**Comment by thiunuwan** [comment]
Go to your repository: Open GitHub and navigate to your pyspeedmath repository.

#### Access the repository settings:

On the repository's main page, look for a gear icon on the right side of the page.
#### Edit the description:

Click on the gear icon. This should open a small panel where you can directly edit the repository description and the URL (if applicable).
Save changes: Once you have corrected the description, it will be automatically saved.

( If the gear icon is not visible or you still can't find the settings, ensure that you have the necessary permissions to make changes to the repository. )

![abcd](https://github.com/user-attachments/assets/6b281dd6-e83b-40c3-afe9-94de3a449f1e)

**Reply by thiunuwan** [reply]
https://github.com/orgs/community/discussions/132595#discussioncomment-10079790

**Reply by ghost** [reply]
Im sorry, but I still cant find it! Please give more info, i'm trying!

**Reply by ghost** [reply]
I thought you meant the repo settings! Thanks for your help!!!!

**Reply by zuczek255** [reply]
Git

**Comment by frank-mendez** [comment]
To change the description of a GitHub repository and format it in Markdown, follow these steps:

Navigate to Your Repository: Go to GitHub and navigate to the repository you want to edit.

Access Repository Settings: Click on the Settings tab located on the right side of the repository menu.

Edit Repository Details:

In the "General" section, you'll find a field labeled "Description".
Enter your new description here.
Save Changes: Scroll down and click the Save changes button.

**Comment by ghost** [comment]
Repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo repo.
